docs/source/performance.md

```diff
@@ -213,74 +209,74 @@ Suffice to say large payload sizes are an anti-pattern in any blockchain solutio
 
 ### Resources
 
-Ensure you monitor the resources of the CouchDB instances, as the larger the state database becomes the more resources CouchDB will consume.
+Ensure monitoring of resources used by CouchDB instances, because as the size of the state database increases, CouchDB consumes more resources.
 
 ### CouchDB cache
 
-When using external CouchDB state database, read delays during endorsement and validation phases have historically been a performance bottleneck. In Fabric v2.x, the peer cache replaces many of these expensive lookups with fast local cache reads.
+When using an external CouchDB state database, read delays during the endorsement and validation transaction phases have shown to be a performance bottleneck. In Fabric v2.x, the peer cache replaces many of these expensive lookups with fast local cache reads.
 
 The cache will not improve performance of JSON queries.
 
-See `CouchDB Cache setting` in the Peer Considerations section for information on configuring the cache.
+See the **CouchDB Cache setting** details above (in **Peer Considerations**) for information on configuring the cache.
 
 ### Indexes
 
-Include CouchDB indexes along with any chaincode that utilises CouchDB JSON queries. Test queries to ensure indexes will be utilised, and avoid queries that can't use indexes. For example use of the query operators $or, $in, $regex result in full data scans. See [Hyperledger Fabric Good Practices For Queries](./couchdb_as_state_database.html#good-practices-for-queries).
+Include CouchDB indexes with any chaincode that utilizes CouchDB JSON queries. Test queries to ensure that indexes are utilized, and avoid queries that cannot use indexes. For example, use of the query operators $or, $in, $regex result in full data scans. See details in [Hyperledger Fabric Good Practices For Queries](./couchdb_as_state_database.html#good-practices-for-queries).
 
-Optimise your queries, complex queries will take more time even with indexing. Ensure your queries result in a bounded set of data. Remember Fabric may also limit the total number of results returned.
+Optimize your queries, because complex queries will take more time even with indexing. Ensure that your queries result in a bounded set of data. Hyperledger Fabric may also limit the total number of results returned.
 
-You can check peer and CouchDB logs to see how long queries are taking and also whether a query was unable to use an index from warning log entries stating the query "should be indexed".
+You can check the peer and CouchDB logs to see how long queries are taking, and whether any query was unable to use an index from a warning log entry stating the query "should be indexed".
 
-If queries are taking a long time, you could try to increase CPU/Memory available to CouchDB or use faster storage.
+If queries are taking a long time, you can increase the CPU/Memory available to CouchDB or use faster storage.
```

Comment:
This implies it will definitely help, but we don't know for sure depending on setup/scenario etc. Which is why it originally was worded as `you can try`

Reply:
sure, the trying applies to speeding up, not to the increasing memory.

Reply:
"If queries are taking a long time, increasing the CPU/Memory available to CouchDB or using faster storage may improve their performance."